test/browser/components.js

```diff
@@ -214,31 +214,31 @@ describe('Components', () => {
 
 		let good, bad;
 		let root = render(<GoodContainer ref={c=>good=c} />, scratch);
-		expect(scratch.innerText, 'new component with key present').to.equal('A\nB');
+		expect(scratch.textContent, 'new component with key present').to.equal('AB');
```

Comment:
I wonder why the newline disappeared here?  I guess we'll merge and see how SauceLabs fares across the supported browsers.